packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -178,7 +178,7 @@ export type DataGridFeature<
      */
     actions?: (column: PublicNormalizedColumnConfig<TEntry>) => {
       iconComponent?: Component | { iconComponent: Component; position?: string };
```

Comment:
Consider renaming and changing type:
Also please always add a tsdoc.
```suggestion
      iconComponent?: Component | 
        { 
          iconComponent: Component;
          /**
           * Will force the iconcomponent to be always shown in the header and not be put into the menu
           */
          alwaysShowInHeader?: boolean
        };
```